spec/factories/countries.rb

```diff
@@ -22,16 +23,11 @@
     sequence :code, Country::CODES.cycle
 
     trait :requires_secondary_education_teaching_qualification do
```

Comment:
I wonder if we could look at renaming these traits to match the new naming convention we're going with `subject_limited`.